src/components/list/listitem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
```

Comment:
I'd recommend rolling this into `<list-container>` so that you can be consistent around creating the dialog and saving it.

If you keep it as-is, all the notes from above about using store data directly, leaving saving to the editor, and leaving serialization to the store apply.

Reply:
Let's put this on the refactor list.